packages/calling-component-bindings/src/hooks/usePropsFor.ts

```diff
@@ -82,5 +89,6 @@ const findSelector = (component: (props: any) => JSX.Element | undefined): any =
     case EndCallButton:
       return emptySelector;
   }
-  throw 'Can\'t find corresponding selector for this component. Please check the supported components from Azure Communication UI Feature Component List.';
+  return undefined;
+  // throw 'Can\'t find corresponding selector for this component. Please check the supported components from Azure Communication UI Feature Component List.';
```

Comment:
This intended to be put back in or removed?

Reply:
This should be removed, this will be one of the internal implementations of usePropsFor, which means error handling will happen in top layer (if you look at the mergedHook.ts)